src/typeDefinitions/discordMessage.types.d.ts

```diff
@@ -17,6 +17,10 @@ export interface messageRequestDataOptions {
 }
 
 export interface messageRequestMember {
+  user: memberUser;
+}
+
+export interface memberUser {
```

Comment:
How about naming it as `member` only?
Also, can't see the usage of these interfaces

Reply:
Check out the complete file there is a complex type of discordMessageRequest that uses these simplified types.

Reply:
Keeping it just `member` might not be a good way as it is related to discord user/member id received in message so i renamed it to `messageMember`